Taskfile.yml

```diff
@@ -97,10 +97,12 @@ tasks:
         silent: true
     cmds:
       - cmd: |
-          if test -z "$(deadcode -test ./...)"; then
+          output=$(deadcode -test ./...)
```

Comment:
This change captures the output of the deadcode command. Previously it would tell if there is a problem, but not what the problem is.